### PR TITLE
[Feature][JDBC-connector] support Jdbc oracle

### PR DESCRIPTION
**Pull request by liugddx**
## Check list

* [x] Code changed are covered with tests, or it does not need tests for reason:
* [x] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [x] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -302,6 +303,12 @@
                 <scope>test</scope>
             </dependency>
 
+            <dependency>
+                <groupId>com.github.noraui</groupId>
+                <artifactId>ojdbc8</artifactId>
+                <version>${oracle.version}</version>
+            </dependency>
```
Comment (CalvinKirs) [inline pom.xml]: This doesn't seem to be an official driver.

Comment (CalvinKirs) [reply pom.xml]: why don't you use  `com.oracle.database.jdbc/ojdbc`?

File: seatunnel-connectors-v2/connector-jdbc/pom.xml [hunk 2]
```diff
@@ -48,6 +48,11 @@
             <artifactId>postgresql</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId></groupId>
+            <artifactId>orac</artifactId>
```
Comment (hailin0) [inline seatunnel-connectors-v2/connector-jdbc/pom.xml]: please check this

File: seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/oracle/OracleJdbcRowConverter.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright @ 2022科大讯飞。 All rights reserved.
+ *
+ * @author: gdliu3
+ * @Date: 2022/8/20 16:27
+ */
```
Comment (hailin0) [inline seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/oracle/OracleJdbcRowConverter.java]: Use apache license？  please check this

File: seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/oracle/OracleTypeMapper.java [hunk 4]
```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright @ 2022科大讯飞。 All rights reserved.
+ *
+ * @author: gdliu3
+ * @Date: 2022/8/20 16:28
+ */
```
Comment (hailin0) [inline seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/oracle/OracleTypeMapper.java]: please check this

**Review by hailin0 — COMMENTED** [review]
please add source and sink e2e-testcase(flink、spark).  reference: #2429 

Please complete the jdbc documentation. [here](https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2)

File: seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/oracle/OracleTypeMapper.java (hunk 4 above)
Comment (liugddx) [reply seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/oracle/OracleTypeMapper.java]: I'm sorry,I'll change it now

Comment (CalvinKirs) [reply seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/oracle/OracleTypeMapper.java]: I have doubts about the license of this code, is this a company code or yours?

Comment (liugddx) [reply seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/oracle/OracleTypeMapper.java]: > 

This is my code,My ide automatically brings out comments,I forgot to replace

File: pom.xml [hunk 5]
```diff
@@ -302,6 +303,12 @@
                 <scope>test</scope>
             </dependency>
 
+            <dependency>
+                <groupId>com.oracle.database.jdbc</groupId>
+                <artifactId>ojdbc8</artifactId>
+                <version>${oracle.version}</version>
+            </dependency>
```
Comment (CalvinKirs) [inline pom.xml]: It's better add `<scope>provide</scope>`

File: seatunnel-e2e/seatunnel-flink-new-connector-e2e/src/test/java/org/apache/seatunnel/e2e/flink/oracle/FakeSourceToOracleIT.java [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+import java.io.IOException;
+
+
+public class FakeSourceToOracleIT extends FlinkContainer {
+	@Test
+	@SuppressWarnings("magicnumber")
+	public void testFakeSourceToOracleSink() throws IOException, InterruptedException {
+		Container.ExecResult execResult = executeSeaTunnelFlinkJob("/oracle/fakesource_to_oracle.conf");
+		Assert.assertEquals(0, execResult.getExitCode());
```
Comment (ic4y) [inline seatunnel-e2e/seatunnel-flink-new-connector-e2e/src/test/java/org/apache/seatunnel/e2e/flink/oracle/FakeSourceToOracleIT.java]: Need to test sink and source on flink and spark.
And need to check the data size and data type. Also need to test exactly-once mode if XA transaction are supported.

**Comment by ic4y** [comment]
@liugddx why close this pr ？

**Comment by liugddx** [comment]
> @liugddx why close this pr ？

I forgot to rebase,The submission record is messed up

**Comment by liugddx** [comment]
> > @liugddx why close this pr ？
> 
> I forgot to rebase,The submission record is messed up

I will resubmit the pr

**Pull request closed without merging**